### DISCUSSION TITLE
No More Hierarchy?

### DISCUSSION THREAD

**Discussion by DreamBagAttic** [keiwando/toonsquid#668] [Q&A]
All my Transform Hierarchies stopped working! 🤷‍♂️🤷‍♂️ Please help. None of my current projects or old projects have transformation hierarchies that work anymore. They are all still there visually but not functionally... all that I was working on had to come to an abrupt halt. Is there something I'm missing?
I updated to the new iOS on the newest iPad (that I purchased for ToonSquid 😅) / and got the latest version of ToonSquid but I'm not sure how to troubleshoot what used to be working 🤷‍♂️ and that seems to be when the problems started

**Comment by kavlor** [comment]
In what way have they stopped working. Are they not behaving as Hierarchies or you can't interact with them at all? Are they behaving like individual objects.

Uploading a video might reveal a lot more or just give more information, the more the better.

Check they are not in a Symbol as you lose the Hiercarchy interaction till you go to 'edit clip'. I can't think of anything else you might 've missed but certainly I find they still work with my projects.

**Reply by DreamBagAttic** [reply]
They completely stopped working. Visually they are still there but when I grab a character, they are fragmented into layers. So to move one character to a new spot I've got to move like all 30 layers individually. The hierarchy looks like it's still there, but the character functions as if it's not... like the hierarchy got turned off or something, though it's worked many times before... but it stopped working on all my hierarchies on all my projects

**Reply by DreamBagAttic** [reply]
I finally back up ALL my files, deleted ToonSquid, reinstalled ToonSquid, started importing the files and hierarchies seem to be working again. If anyone has ideas on this, I'd like to avoid it happening in the future - quite a pain. Thx. - but I did start playing with procreate dreams. Seems cool, but I honestly like ToonSquid better...

**Reply by keiwando** [reply]
Are the drawing layers that the character is made up of [pixel layers](https://toonsquid.com/handbook/layers/pixel_layer/)?

**Reply by DreamBagAttic** [reply]
> Are the drawing layers that the character is made up of [pixel layers](https://toonsquid.com/handbook/layers/pixel_layer/)?

No. All vector layers. (Thanks for your response)

**Reply by keiwando** [reply]
Then the most likely cause is that you might have (unintentionally) enabled the [Edit path](https://toonsquid.com/handbook/transform/transform_tool/#tool-options) option in the transform tool, so that the transform tool was editing the path control points instead of the layer transforms.

**Reply by DreamBagAttic** [reply]
> Then the most likely cause is that you might have (unintentionally) enabled the [Edit path](https://toonsquid.com/handbook/transform/transform_tool/#tool-options) option in the transform tool, so that the transform tool was editing the path control points instead of the layer transforms.

Oh dang. That probably is what I did. I couldn't figure it out for the life of me. Thank you. I really love this app
